monai/apps/pathology/transforms/spatial/array.py

```diff
@@ -17,6 +17,7 @@
 
 from monai.config.type_definitions import NdarrayOrTensor
 from monai.transforms.transform import Randomizable, Transform
+from monai.utils import convert_data_type, convert_to_dst_type
 from monai.utils.enums import TransformBackends
 
 __all__ = ["SplitOnGrid", "TileOnGrid"]
```

Comment:
looks like `SplitOnGrid(2, 2)(torch.arange(12).reshape(1, 3, 4))` works fine but `SplitOnGrid(2, 2)(np.arange(12).reshape(1, 3, 4))` doesn't work, could you help fix it here? #3378 was merged too quickly

Reply:
@wyli, sorry I missed this. I will fix it now.

Reply:
https://github.com/Project-MONAI/MONAI/pull/3386